plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions.php

```diff
@@ -31,6 +31,23 @@ protected function enqueue_data( array $attributes = [] ) {
 		parent::enqueue_data( $attributes );
 		$this->asset_data_registry->add( 'isBlockifiedAddToCart', Features::is_enabled( 'blockified-add-to-cart' ) );
 		$this->asset_data_registry->add( 'productTypes', wc_get_product_types() );
+		$this->asset_data_registry->add( 'themeTemplatePartPaths', $this->get_theme_template_part_paths() );
```

Comment:
Besides changing `paths` for `ids` in this variable, I would also mention the block name. Something like:

```suggestion
		$this->asset_data_registry->add( 'addToCartWithOptionsTemplatePartIds', $this->get_theme_template_part_paths() );
```